examples/mapreduce/job.properties

```diff
@@ -39,3 +39,5 @@ oozie.use.system.libpath=true
 userName=${user.name}
 oozie.wf.application.path=${nameNode}/user/${user.name}/${examplesRoot}/apps/mapreduce
 outputDir=output
+oozie.wf.workflow.notification.url=http://35.234.110.167:8080/workflow?job-id=$jobId&status=$status
```

Comment:
```suggestion
oozie.wf.workflow.notification.url=http://www.example.com:8080/workflow?job-id=$jobId&status=$status
```

Reply:
Fixed in 57fd9a185f161ca802aa7aa6e158f4132b911c70